google/ggrc-core
# GGRC-7648 Rework test_import_issues.py

# Issue description

Rework test_import_issues.py

# Steps to test the changes

Execute test/integration/ggrc/converters/test_import_issues.py and make sure the tests pass without allowing usage of custom slugs

# Solution description

Remove custom slug from import_data in the following methods
test_import_empty_mandatory
test_import_with_mandatory
test_import_without_mandatory

# Sanity checklist




# PR Review checklist
